hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -672,31 +672,67 @@ tests:
 - interval: 1m
   input_series:
     - series: 'kubevirt_hco_node_status_max_images{node="node1"}'
-      values: '50+0x10 -1'
+      values: '50+0x20 -1+0x10 50+0x35'
     - series: 'kubevirt_hco_node_number_of_images{node="node1"}'
-      values: '0+10x5'
+      values: '0+0x10 51+0x55'
+    - series: 'kubevirt_hco_hyperconverged_cr_exists{}'
+      values: '1+0x40 0+0x5 1+0x20'
+    - series: 'kube_node_labels{node="node1", label_kubevirt_io_schedulable="true"}'
+      values: '1+0x60 stale'
+    - series: 'kube_node_labels{node="node1"}'
+      values: 'stale+0x60 1+0x65'
 
   alert_rule_test:
-    # Not yet 5m since kubevirt_hco_node_number_of_images reached 50 (same as kubevirt_hco_node_status_max_images)
-    - eval_time: 9m
+    # node images: 0, max images: 50, no alert
+    - eval_time: 6m
       alertname: NodeStatusMaxImagesExceeded
       exp_alerts: [ ]
 
-    # 5m since kubevirt_hco_node_number_of_images reached 50 (same as kubevirt_hco_node_status_max_images)
-    - eval_time: 10m
+    # node images: 51, max images: 50, alert
+    - eval_time: 16m
       alertname: NodeStatusMaxImagesExceeded
       exp_alerts:
         - exp_annotations:
-            summary: "Node node1 contains more images than the configured maximum number of images reportable in the node status"
+            summary: "1 node(s) contain more images than the configured maximum number of images reportable in the node status"
             runbook_url: "https://github.com/openshift/runbooks/blob/master/alerts/openshift-virtualization-operator/NodeStatusMaxImagesExceeded.md"
           exp_labels:
             kubernetes_operator_component: "hyperconverged-cluster-operator"
             kubernetes_operator_part_of: "kubevirt"
             operator_health_impact: "warning"
             severity: "warning"
-            node: node1
 
-    # kubevirt_hco_node_status_max_images was set to -1 (unlimited)
-    - eval_time: 11m
+    # node images: 51, max images: -1, no alert
```

Comment:
Please also mention if the HCO CR exists. 
Also O believe you might be missing a test where the CR doesn't exists and the node images: 51, max images: 50